packages/fxa-shared/metrics/amplitude.js

```diff
@@ -103,6 +103,19 @@ function mapDisconnectReason(eventType, eventCategory) {
   }
 }
 
+function mapDomainValidationResult(
+  eventType,
+  eventCategory,
+  eventTarget,
+  data
+) {
+  // This function is called for all fxa_reg event types, only add the event
+  // properties for the results pertaining to domain_validation_result.
+  if (eventType === 'domain_validation_result' && eventCategory) {
```

Comment:
@irrationalagent - if an event, e.g., `domain_validation_result` is emit 3 times with different values for the same event property, will that show up as 3 different events?

Reply:
Sorry I missed this. Yes, it will be three separate events, one event with each event property value.